### DISCUSSION TITLE
[Feature] external and internal URL for accessing immich

### DISCUSSION THREAD

**Discussion by rithask** [immich-app/immich#10063] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, when I want to upload photos to immich when I’m home, I sign out and set the server address to my machine’s local IP address and when I’m outside, I want to re login with immich’s public address.

It would be great if we have an option to set internal and external URL to access immich. Internal URL will be used when I’m at my home connected to a specific SSID(s) mentioned in the immich app and external when I’m outside

This screenshot is from Home Assistant iOS app. 
![IMG_5628](https://github.com/immich-app/immich/assets/74851056/f8138d01-f246-459d-8848-7cb448f02df6)
.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe: #308
